### ISSUE TITLE
Redash on Production

### ISSUE THREAD

**Issue by SimonLab** [emfoundation/ce100-app#1044]
Once the new version of the application is deployed, we need to update Redash to connect the the production URL to allow admin users to get the "real" data information

**Comment by iteles** [comment]
@asimonw and @georgemillard have agreed that this should be promoted to production if they are the only ones with access.

**Comment by SimonLab** [comment]
@asimonw @georgemillard I've connected the production database to Redash, created a query to see the list of users, and added a new dashboard. Let me know if it's working for you, thanks :+1:

**Comment by georgemillard** [comment]
Hi @SimonLab, just working with redash, and the timestamp stored in the last login column of the people table returns what I think is the number of milliseconds since a fixed date. Could you confirm the format so we can decode it into something useful in our reports?

Thanks!

**Comment by SimonLab** [comment]
@georgemillard yes you are right, we are using the Node method ```Date.now()``` to update the last login timestamp:
https://github.com/emfoundation/ce100-app/blob/4f4d83675b555f9246de4d68ff8f5c5cb8505d63/server/handlers/auth/login.js#L53

"The Date.now() method returns the number of milliseconds elapsed since 1 January 1970 00:00:00 UTC"

I'm not entirely sure how to convert a timestamp (in milliseconds) to a readable date format in Python but I can have a better look if needed.
I also notice that Redash adds commas to format the timestamp number, this might be because the format of the column in Postgres for this value is ```bigint```

**Comment by georgemillard** [comment]
Thanks Simon, that's just what I needed! I'll look into it from here.
